### PR TITLE
initial react-native example

### PR DESCRIPTION
**Pull request by pb82**
## Motivation

Add an initial react-native example. Simple Navigation included with only one Welcome page. No SDK usage at the moment because we don't yet have any JS SDKs.

Includes a README to help users get started.
Uses the AeroGear Logo as the App icon.

### CODE REVIEW

**Comment by wtrocki** [comment]
> I would add some Redux boilerplate and some eslint rules.

That is vanilla example - it's not intended to build on. 
It should be minimal as possible.

File: example/react-native/android/app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,150 @@
+apply plugin: "com.android.application"
```
Comment (wtrocki) [inline example/react-native/android/app/build.gradle]: Not sure if we can avoid pushing actual platform source code?

**Comment by josemigallas** [comment]
@wtrocki The files here are the minimum scaffold for a real React Native app. The example you are referencing is generated by [create-react-native-app](https://github.com/react-community/create-react-native-app) which is only meant to be using [Expo](https://expo.io/). I believe this project was created using `react-native init`.

**Comment by pb82** [comment]
@wtrocki @josemigallas yes, the boilerplate was generated using `react-native init`. I would prefer to leave the platform code in the repo. As far as i can see, other popular projects (e.g. https://github.com/iSimar/HackerNews-React-Native, https://github.com/7kfpun/FinanceReactNative, ...) do this too. Otherwise the project would not work with `react-native run-<platform>`

**Comment by pb82** [comment]
@josemigallas Maybe we can add Redux to the example when we have a use case with the SDK where it makes sense (auth).

**Comment by josemigallas** [comment]
Yes, that was just my suggestion, I think this is good already

**Pull request merged**